File: Python/account.py
class Account:
    id = int
    name = str
    document = str
    email = str
    password = str
    
    def __init__(self, name, document, id, email, password ):
        self.name = name
        self.document = document
        self.id = id
        self.email = email
        self.password = password
    

File: PHP/account.php
<?php
class Account{
    public $id;
    public $name;
    public $document;
    public $email;
    public $password;

    public function __construct($name,$document,$id, $email,$password){
        $this->name = $name;
        $this->document = $document;
        $this->id = $id;
        $this->email = $email;
        $this->password = $password;
    }
}